exchange/exchangetest/imp-fpd-multiple-imp.json

```diff
@@ -0,0 +1,294 @@
+                            "imp": {
+                                "appnexus": {
+                                    "audio": {
+                                        "maxduration": 20,
+                                        "durfloors": [{
+                                            "maxdur": 30
+                                        }],
+                                        "poddur": 40
+                                    },
+                                    "native": {
+                                        "ver": "2.2",
+                                        "ctype": 1
+                                    }
+                                }
+                            }
+                        }
+                    }
+                }
+            ]
+        }
+    },
+    "outgoingRequests": {
+        "appnexus": {
+            "expectRequest": {
+                "ortbRequest": {
+                    "id": "some-request-id",
+                    "site": {
+                        "page": "test.somepage.com"
+                    },
+                    "imp": [
+                        {
+                            "id": "imp-id-1",
+                            "banner": {
+                                "w": 100,
+                                "h": 200,
+                                "format": [
+                                    {
+                                        "w": 3000,
+                                        "h": 6000
+                                    }
+                                ],
+                                "btype": [1,2]
+                            },
+                            "ext": {
+                                "bidder": {
+                                    "placementId": 1
+                                }
+                            }
+                        },
+                        {
+                            "id": "imp-id-2",
+                            "video": {
+                                "mimes": [
+                                    "video/mp4"
+                                ],
+                                "w": 200,
+                                "h": 400
+                            },
+                            "ext": {
+                                "bidder": {
+                                    "placementId": 1
+                                }
+                            }
+                        },
+                        {
+                            "id": "imp-id-3",
+                            "audio": {
+                                "mimes": ["audio/mp4"],
+                                "minduration": 1,
+                                "maxduration": 20,
+                                "durfloors": [{
+                                    "mindur": 1,
+                                    "maxdur": 30,
+                                    "bidfloor": 10.0
+                                }],
```

Comment:
Why is the merge function merging objects inside of an array instead of just selecting the overriding array? I was expecting this to be:
```
"durfloors": [{
    "maxdur": 30
}],
```

Reply:
This is a difference between merging approaches from an earlier PR. Good find. Let's address separately.

Reply:
Fixed in https://github.com/prebid/prebid-server/pull/3708.